src/arkode/arkode_butcher_dirk.c

```diff
@@ -600,6 +609,431 @@ ARKodeButcherTable ARKodeButcherTable_LoadDIRK(ARKODE_DIRKTableID imethod)
     B->c[7] = RCONST(1.0);
     break;
 
+  case(ARKODE_ESDIRK324L2SA_4_2_3):   /* ESDIRK3(2)4L[2]SA (A,L stable) */
+#define g   RCONST(0.43586652150845899941601945)
```

Comment:
This is 26 digits but quad precision support should require 34 I think.  I have the coefficients in Mathematica and can update this for quad precision and in a form that only needs `g` and `g2`

Reply:
That would be great, Steven!  I had copied the value from their paper, which obviously did not go out to the 36 or so needed for full quad precision.

Did you happen to re-derive other methods of theirs from this paper?  I ask because multiple of them have typos in the paper (and they have not issued an errata).  For example, ESDIRK6(5)9[2]SA has incorrect $\hat{b}$ vector (the values don't even sum to 1).  I believe there were 3 or 4 methods with errors that I'd wanted to include, so I was imagining that when I had sufficient free time I would rederive those to fix the mistaken coefficients.

Reply:
I have many of the Kennedy and Carpenter methods set up in Integreat from previous projects.  Most were rederived and validated from the list of order conditions and properties listed in the papers.  If there is any issue with a coefficient, it is easy to check if the method satisfies all order conditions.  I recall coming across an incorrect coefficient in ESDIRK5(4)8L[2]SA and one other method.